src/libs/ajax/data-table-providers/WdsDataTableProvider.test.ts

```diff
@@ -515,7 +529,7 @@ describe('WdsDataTableProvider', () => {
   describe('getPage', () => {
     it('restructures a WDS response', () => {
```

Comment:
Looking at the existing structure of this test and the other async method tests, it would be better to shift these to async await pattern on the test method:

```
it('does something', async () => {
  // Arrange
  ...
  // Act
  const result = await myThing.theAsyncMethod(...)
  // Assert
  expect result.toEqual(...)
}  
```

Reply:
I might consider this out of scope for refactoring the test file in this PR/AJ-790 -- happy to do so though in a follow-up PR or ticket though!